tutorials/rest-http-service-with-axum.mdx

```diff
@@ -0,0 +1,357 @@
+---
+title: "Writing a Rest HTTP Service with Axum"
+---
+
+In this guide you'll learn the basics of how to write a competent Axum HTTP REST service - first starting off with basic routing and writing functions to act as our endpoints, adding app State and middleware functions, using cookies and CORS, then finally looking at testing our app.
+
+### Getting Started
+
+To get started, you'll want to initialise a Shuttle project with Axum (you can find more about getting started [here](https://docs.shuttle.rs/introduction/installation)).
+
+```rust
+cargo shuttle init --axum
+```
+
+This will initialise a project with a basic router and basic configuration so that it can be deployed immediately.
+
+## Routing
+
+In Axum, routers can be made by writing `Router::new()` to create it and then adding a route with the `route` method by chaining it. Below is an example that returns a router that has one route at the base that will return "Hello World!" if we go to `localhost:8000`.
+
+```rust
+#[shuttle_runtime::main]
+async fn axum() -> shuttle_axum::ShuttleAxum {
+
+let router = Router::new().route("/", get(hello_world));
+
+Ok(router.into())
+}
+
+// this is a function that returns a static string
+// all functions used as endpoints must return a HTTP-compatible response
```

Comment:
It looks like the rest of the docs use non-capitalised sentences. I wouldn't mind amending this but it would probably require maybe changing the rest of the docs for consistency

Reply:
No worries!